### ISSUE TITLE
how to compute test accuracy

### ISSUE THREAD

**Issue by Victoria2333** [facebookresearch/clevr-iep#25]
I see that answers in CLEVR are only for train and validation set,how to get ground-truth answers to be compared with our predicted answers??I couldn't find test.py in your Read.me file,there is only predicted answer.Or I need to upload my answer json file to CLEVR server or something else??
Looking forward to your reply,it will be very helpful for me~~
Thank you very much~~~

**Comment by Victoria2333** [comment]
I created a json file with question_id and predicted answer,so i just send the file to jcjohnson's email???

**Comment by Victoria2333** [comment]
@rbgirshick ,I created a json file with question_id and predicted answer,so i just send the file to jcjohnson's email???

**Comment by rbgirshick** [comment]
We haven't set up an evaluation server, so currently the protocol is to email results to @jcjohnson.

**Comment by Victoria2333** [comment]
@rbgirshick,do you know johnson's email???I couldn't find in his essays...
Thank you for your attention~~

**Comment by Victoria2333** [comment]
@rbgirshick
Is his email jcjohns@cs.stanford.edu??Usually how long will it take for him to reply???
Thank for your attention~~

**Comment by Victoria2333** [comment]
@rbgirshick 
Usually how long will it take for him to reply???
Looking forward to your reply~~~

**Comment by gaopeng-eugene** [comment]
@Victoria2333 Do you get the test accuracy successfully? If so, can you share experience on how to formalize the test json file?
